PactSafe iOS Demo/Controllers/LoginViewController.swift

```diff
@@ -6,158 +6,154 @@
+    }
+
+    /// Check the status of the specified signer ID within a PactSafe group.
+    ps.signedStatus(for: signerId, groupKey: groupKey) { needsAcceptance, contractIds in
+      if needsAcceptance {
+        self.showContractUpdates(forSignerId: signerId,
+                                 password: passwordText,
+                                 filterContractIds: contractIds)
+      } else {
+        DispatchQueue.main.async {
+          self.segueToHome()
         }
+      }
     }
-    
-    private func showContractUpdates(forSignerId signerId: String,
-                                     password passwordText: String,
-                                     filterContractIds: [String]? = nil) {
-        
-        self.ps.loadGroup(groupKey: self.groupKey) { (groupData, error) in
-            guard let groupData = groupData, let contractsData = groupData.contractData else { return }
-            self.psGroupData = groupData
-            
-            var titlesOfContracts = [String]()
-            
-            var updatedContractsMessage: String = "We've updated the following: "
-            
-            if let cidsFilter = filterContractIds {
-                contractsData.forEach { (key, value) in
-                    if cidsFilter.contains(key) { titlesOfContracts.append(value.title) }
-                }
-            } else {
-                contractsData.forEach { (key, value) in
-                    titlesOfContracts.append(value.title)
-                }
-            }
-            
-            let contractTitles = titlesOfContracts.map { String($0) }.joined(separator: " and ")
-            updatedContractsMessage.append(contractTitles)
-            updatedContractsMessage.append(".\n \n Please agree to these changes.")
-            
-            DispatchQueue.main.async {
-                let alert = self.updatedTermsAlert("Updated Terms", message: updatedContractsMessage, email: signerId, password: passwordText)
-                self.present(alert, animated: true, completion: nil)
-            }
+  }
+
+  private func showContractUpdates(forSignerId signerId: String,
+                                   password passwordText: String,
+                                   filterContractIds: [String]? = nil) {
+    ps.loadGroup(groupKey: groupKey) { groupData, _ in
+      guard let groupData = groupData, let contractsData = groupData.contractData else { return }
+      self.psGroupData = groupData
+
+      var titlesOfContracts = [String]()
+
+      var updatedContractsMessage: String = "We've updated the following: "
+
+      if let cidsFilter = filterContractIds {
+        contractsData.forEach { key, value in
+          if cidsFilter.contains(key) { titlesOfContracts.append(value.title) }
+        }
+      } else {
+        contractsData.forEach { _, value in
+          titlesOfContracts.append(value.title)
         }
+      }
+
+      let contractTitles = titlesOfContracts.map { String($0) }.joined(separator: " and ")
```

Comment:
what happens if you have more than two here? will they all be joined with "this and that and those"?

Reply:
Indeed. In real-world usage, it would likely have to be altered. Haven't been able to dig far enough into the best approaches for something like this.